src/components/file-input/examples/file-input-type-filtering.tsx

```diff
@@ -0,0 +1,81 @@
+import { FileInfo } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Example of a file input component with type filtering
+ */
+@Component({
+    tag: 'limel-example-file-input-type-filtering',
+    shadow: true,
+})
+export class FileInputTypeFilteringExample {
+    @State()
+    private disabled = false;
+
+    @State()
+    private readonly = false;
+
+    @State()
+    private multiple = false;
+
+    @State()
+    private files: FileInfo[] = [];
+
+    public render() {
+        return [
+            <limel-file-input
+                onFilesSelected={this.handleFilesSelected}
+                accept="image/*"
+                disabled={this.disabled || this.readonly}
+                multiple={this.multiple}
+            >
+                <limel-button label="Select an image" />
+                {this.files.map((file) => (
+                    <limel-chip
+                        text={file.filename}
+                        icon={file.icon}
+                        disabled={this.disabled}
+                        readonly={this.readonly}
+                    />
+                ))}
+            </limel-file-input>,
+            <limel-example-controls>
+                <limel-checkbox
+                    checked={this.disabled}
+                    label="Disabled"
+                    onChange={this.setDisabled}
+                />
+                <limel-checkbox
+                    checked={this.readonly}
+                    label="Readonly"
+                    onChange={this.setReadonly}
+                />
+                <limel-checkbox
+                    checked={this.readonly}
+                    label="Multiple"
```

Comment:
I admit I initially was a bit confused by this label 😅, I thought it was about limiting the number of selected files to one.